es-ds-components/components/es-dropdown.vue

```diff
@@ -0,0 +1,188 @@
+<script setup lang="ts">
+import Dropdown from 'primevue/dropdown';
+
+interface Props {
+    disabled?: boolean;
+    modelValue?: any;
+    options?: string[] | { label: string; value: string }[];
+    placeholder?: string;
+    label?: string;
+}
+const props = withDefaults(defineProps<Props>(), {
+    disabled: false,
+    modelValue: undefined,
+    options: () => [],
+    placeholder: '',
+    label: '',
+});
+
+const emit = defineEmits(['update:modelValue']);
+
+const isOpen = ref(false);
+const isFocused = ref(false);
+
+const isSelected = (option: any) => option === props.modelValue;
+
+const id = useId();
+</script>
+
+<template>
+    <div
+        class="input-wrapper justify-content-end"
+        :class="$attrs.class">
+        <label
+            v-if="label !== ''"
+            :for="id">
+            {{ label }}
+        </label>
+        <label
+            v-else
+            :for="id"
+            class="sr-only">
+            Select an option
+        </label>
+        <dropdown
+            :input-id="id"
+            :class="[
+                'es-dropdown-input d-flex align-items-center bg-white rounded-xs justify-content-between p-100',
+                {
+                    disabled: disabled,
+                    'es-dropdown-open': isOpen,
+                    focused: isFocused && !isOpen,
+                },
+            ]"
+            :disabled="disabled"
+            :focus-on-hover="false"
+            :model-value="modelValue"
+            :option-label="options.length > 0 && typeof options[0] === 'object' ? 'label' : undefined"
+            :options="options"
+            :placeholder="placeholder"
+            :pt="{
+                panel: { class: 'es-dropdown-panel bg-white rounded-xs' },
+                wrapper: { class: 'es-dropdown-wrapper' },
+                list: { class: 'p-0 m-0 list-unstyled' },
+                item: {
+                    class: [
+                        'es-dropdown-item d-flex justify-content-between p-100 pl-200',
+                        {
+                            'input-focused': isFocused,
```

Comment:
Something about this logic is causing this to happen: when you use the mouse to click on the dropdown, opening the menu, then Command-Tab to another window, then Command-Tab back, the item has a focus outline when it shouldn't.

https://github.com/user-attachments/assets/6be598ce-98a7-4f35-93fb-db49ed91fb66

As a general rule, I don't think the user should ever see these ugly focus outlines unless they're tabbing or arrowing  through via keyboard.